tests/cube/draft/createPacks.test.ts

```diff
@@ -0,0 +1,448 @@
+import seedrandom from 'seedrandom';
```

Comment:
Even though createPacks lives in the createdraft.ts file, it felt more logical to me to name the test after the function